properties.schema

```diff
@@ -111,21 +124,140 @@
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Defines the Item text / icon colour when selected. Should be a colour that provides good contrast against the Item selected colour."
+            "help": "Defines the item text / icon colour when selected. Should be a colour that provides good contrast against the item selected colour."
           },
           "visited": {
             "title": "Visited colour",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Defines the Item colour when it has been visited."
+            "help": "Defines the item colour when it has been visited."
           },
           "visited-inverted": {
             "title": "Visited colour - inverted",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Defines the Item text / icon colour when visited. Should be a colour that provides good contrast against the Item visited colour."
+            "help": "Defines the item text / icon colour when visited. Should be a colour that provides good contrast against the item visited colour."
+          },
+          "item-color-disabled": {
+            "title": "Item colour - disabled",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": ""
+          },
+          "item-color-inverted-disabled": {
+            "title": "Item colour - inverted disabled",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Defines the item text / icon colour when disabled. Should be a colour that provides good contrast against the item disabled colour."
+          }
+        }
+      },
+
+      "_itemsUi": {
+        "type": "object",
+        "required": false,
+        "title": "Content Items UI (action not content)",
+        "properties": {
+          "item-ui-color": {
+            "title": "Item UI colour",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Defines the clickable areas in components that don't contain course content."
```

Comment:
minor but 'selectable' is more inclusive word choice than 'clickable'

Reply:
This should now be resolved